Aztec/Classes/TextKit/TextView.swift

```diff
@@ -605,6 +644,37 @@ open class TextView: UITextView {
         return false
     }
 
+    private func shouldRemoveSingleLineParagraphAttributes(insertedText text: String, at location: Int) -> Bool {
```

Comment:
I have to admit I'm not able to quicly grasp why this is necessary by just reading the code.

Can you document this method, including information and possibly an example of why this is needed?